### ISSUE TITLE
Contributing Datasets: flags “New!” and “Updated!” are missing with GSDs

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/portal#40]
**Contributing Datasets**
https://www.dev.catalogue.life/data/contributing-datasets.html 

- [ ] Flags “New!” and “Updated!” are missing with GSDs.
Example: http://www.catalogueoflife.org/annual-checklist/2019/info/databases  

I guess, provision for this field is missing in metadata as well.

Perhaps @gdower may explain how it works.

**Comment by mdoering** [comment]
we should be able to compare the last syncs between releases and thereby detect what has changed.

**Comment by mdoering** [comment]
... but it probably needs an API for that

**Comment by yroskov** [comment]
“New!” and “Updated!” flags are messages from editor to users regarding GSDs. They cannot be automated. 

However, if your software will be able to notify users via API about all modified taxonomic records, it will be good step forward.

**Comment by gdower** [comment]
In the past the statuses were:

0 = not new and not updated since previous annual edition
1 = new dataset added since previous annual edition
2 = not new and updated since previous annual edition

**Comment by mdoering** [comment]
I still think this should be automated, why not?

**Comment by gdower** [comment]
I agree it would be good to automate it because otherwise the update flags can get out of sync. It might be relatively easy because the update flags just need to be reset to 0 when the annual edition is released.

**Comment by mdoering** [comment]
I would compare the sync attempt from the last release with the current sync attempt. If that has changed we can flag it as updated. If it was not even present as new

**Comment by mdoering** [comment]
as we do not yet have an annual release I would flag those that have changed since we did the first monthly release?

**Comment by yroskov** [comment]
@gdower Automated?
How we'll deal with cases when we did not update WPlants with a new version but only corrected bugs left in previous months? I do not want to expose that database as Updated until we get a new version. 
How we'll deal with Alucitoidea? It is New for CoL2020, but it will be updated each month. 
Keep this flag manual.

**Comment by yroskov** [comment]
> I would compare the sync attempt from the last release with the current sync attempt. If that has changed we can flag it as updated. If it was not even present as new

Do this for every change taxonomic record, but not for entire GSD. 

Automatic flag will confuse users due to changes caused by bugs in currently unstable software.

**Comment by mdoering** [comment]
why not inform users that data has changed even if the underlying dataset might not have changed but it was just a fix or new editorial decision? The more manual flags the more inconsistencies and work to do
